eng/testing/WasmRunnerTemplate.sh

```diff
@@ -1,19 +1,20 @@
 #!/usr/bin/env bash
 
 EXECUTION_DIR=$(dirname $0)
-JAVASCRIPT_ENGINE=$1
 
 cd $EXECUTION_DIR
 
 XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 
-dotnet xharness wasm test --engine=$JAVASCRIPT_ENGINE \
-    --js-file=runtime.js -v \
-    --output-directory=$XHARNESS_OUT \
-    -- --enable-gc --run WasmTestRunner.dll ${@:2}
+if [ ! -x "$(command -v xharness)" ]; then
+	HARNESS_RUNNER="dotnet"
+fi
+
+# RunCommands defined in tests.mobile.targets
+[[RunCommands]]
 
 _exitCode=$?
 
-echo "Xharness artifacts: $XHARNESS_OUT"
+echo "Xharness artifacts: `ls -lh $XHARNESS_OUT`"
```

Comment:
why do we need this? I actually want to get the directory and not which files, or at least not in a format like this:

```
  XHarness exit code: 1001
  Xharness artifacts: total 64
  -rw-r--r--  1 alexander  staff    29K Jun 18 18:32 testResults.xml
```

Reply:
Tbh, maybe having `set -x` at the beginning would do the trick much better?

Reply:
@premun how would that help? this is mostly for local runs where a human looks at the output and I want to Ctrl/Cmd+Click on the path in my terminal to open the folder/file.